Source/Core/VideoCommon/VertexLoaderBase.cpp

```diff
@@ -91,6 +97,62 @@ class VertexLoaderTester : public VertexLoaderBase
   std::vector<u8> buffer_b;
 };
 
+u32 VertexLoaderBase::GetVertexSize(const TVtxDesc& vtx_desc, const VAT& vtx_attr)
+{
+  u32 size = 0;
+  if (vtx_desc.low.PosMatIdx)
+    size++;
+  for (auto texmtxidx : vtx_desc.low.TexMatIdx)
```

Comment:
Can you not do `size += vtx_desc.low.TexMatIdx.size();`?

Reply:
That would always add 8, but we want to count the number that are set.